language/move-compiler/src/naming/ast.rs

```diff
@@ -223,16 +226,19 @@ pub enum Exp_ {
     ModuleCall(
         ModuleIdent,
         FunctionName,
+        bool,
         Option<Vec<Type>>,
         Spanned<Vec<Exp>>,
     ),
+    LocalCall(Var, Spanned<Vec<Exp>>),
```

Comment:
Thoughts on making this `LambdaCall` or something? I get it that its a call on a local variable, but I thought it was a reference to scoping the first time I saw it

Reply:
It seems a bit odd because we are not calling a lambda but a named value. I renamed it to `VarCall`.